hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -140,7 +139,6 @@ public static Versioning getVersioning(boolean versioning) {
   }
 
   public static final String DELETING_KEY_PREFIX = "#deleting#";
-  public static final String DELETED_KEY_PREFIX = "#deleted#";
   public static final String DELETE_TRANSACTION_KEY_PREFIX = "#delTX#";
   public static final String BLOCK_COMMIT_SEQUENCE_ID_PREFIX = "#BCSID";
```

Comment:
Rename, because this is not actually a prefix, but a piece of metadata. Also make sure it is placed in the metadata table when used.